packages/models-library/src/models_library/clusters.py

```diff
@@ -159,6 +159,7 @@ class Cluster(BaseCluster):
     id: ClusterID = Field(..., description="The cluster ID")
 
     model_config = ConfigDict(
+        extra="allow",
```

Comment:
why is this necessary?

Reply:
https://github.com/ITISFoundation/osparc-simcore/pull/6584#discussion_r1828945148